client/src/pages/private/ParticipantEOI.js

```diff
@@ -267,55 +267,50 @@ export default () => {
           </div>
         )}
         {participant && (
-          <Grid justify='center' container>
-            <Grid container direction='row'>
-              <Grid item xs={6}>
-                <Typography variant='h4'>Participant Express of Interest</Typography>
-              </Grid>
-              <Grid item xs={6}>
-                <Grid container spacing={2} justify='flex-end'>
-                  <Grid item>
-                    <Button variant='contained' onClick={onEdit}>
-                      {!enableEdit ? 'Edit Info' : 'Done Edit'}
-                    </Button>
-                  </Grid>
-                  <Grid item>
-                    <Button
-                      disabled={disableWithdraw}
-                      variant='contained'
-                      onClick={() => {
-                        setOpenConfirmInterest(true);
-                      }}
-                    >
-                      Reconfirm interest
-                    </Button>
-                  </Grid>
-                  <Grid item>
-                    <DeleteButton
-                      disabled={disableWithdraw}
-                      variant='contained'
-                      color='primary'
-                      onClick={() => setOpenWithdraw(true)}
-                    >
-                      Withdraw PEOI
-                    </DeleteButton>
-                  </Grid>
-                </Grid>
-              </Grid>
-            </Grid>
-            <Grid item xs={12} sm={11} md={10} lg={8} xl={6}>
-              {/** Form */}
-              <Form
-                initialValues={participant.body}
-                isDisabled={!enableEdit}
-                hideSummary={true}
-                showIdentityQuestions
-                enableFields={['phoneNumber', 'postalCode']}
-                onSubmit={onFormSubmit}
-                editMode={true}
-                isSubmitted
-              />
-            </Grid>
+          <Grid item xs={12} sm={11} md={10} lg={8} xl={6}>
```

Comment:
I think, there will be possible conflict with PR-#577

Reply:
This PR is branched off that one so it should work out

Reply:
Ah I had the wrong base, my bad